image.yaml

```diff
@@ -0,0 +1,31 @@
+name: "jboss-processserver-6/processserver64-openshift"
+version: "1.1"
+description: "Platform for executing business rules on JBoss BPMS Intelligent Process Server 6.4."
```

Comment:
Use our schema for this key: "Red Hat JBoss BPMS Intelligent Process Server 6.4 OpenShift container image"